libblockverifier/ExecutiveContext.cpp

```diff
@@ -34,6 +34,16 @@ using namespace dev::eth::abi;
 using namespace dev::blockverifier;
 using namespace dev;
 using namespace std;
+
+ExecutiveContext::ExecutiveContext() : m_addressCount(0x10000) {}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Member variable 'ExecutiveContext::m_blockInfo' is not initialized in the constructor.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5547730)